core/templates/dev/head/pages/exploration_player/ExplorationPlayerStateService.js

```diff
@@ -28,14 +28,23 @@ oppia.factory('ExplorationPlayerStateService', [function() {
     },
 
     isInteractionInline: function(stateName) {
+      if (stateName === null) {
```

Comment:
The changes in this file are probably OK. But if there's a way to prevent these functions being called at all when we're in "concept card" mode, that would be even better.

Reply:
Yeah, will add the above created function before all these functions are called.

Reply:
Done.